### ISSUE TITLE
Jakarta EE 9.1 alignment

### ISSUE THREAD

**Issue by Emily-Jiang** [eclipse/microprofile-rest-client#316]
### For each spec, the following updates need to happen, refer [this doc](https://docs.google.com/document/d/1bhhfmhxHJpQdmdUtQw5o_WtScpzkfHoTNGPvrQnGhc0/edit) for instructions:

- [ ] Create a branch called `jakarta91-branch` off master to perform Jakarta EE 9.1 alignment
- [ ] Only jakarta namespace changes are allowed in the release without any other functional changes with the exception of typos or tck fixes
- [ ] spec docs: rename Java EE to Jakarta EE, rename javax to jakarta: `find . -type f -name '*.asciidoc' -exec sed -i '' s/javax./jakarta./ {} +`
- [ ] Update all javax to jakarta for all java files `import javax.*` needs to be updated to `jakarta.*`: `find . -type f -name '*.java' -exec sed -i '' s/javax./jakarta./ {} +`
- - [ ] TCKs
- - [ ] APIs
- [ ] Utilise microprofile-parent 2.1 as it has pulled in all of the Jakarta EE 9.1 as well as the necessary TCK
- [ ] Update bnd file:  `find . -type f -name ‘*.bnd' -exec sed -i '' s/javax./jakarta./ {} +`

- [ ] service loader file javax.* to jakarta.*: `find . -type f -name "javax*" -exec sh -c 'mv "$0" "${0/javax/jakarta}"' '{}' \;
`
- [ ] deliver the changes to both `jakarta91-branch` and `master`
- [ ] Perform a RC release off the branch `jakarta91-branch`

**Comment by Emily-Jiang** [comment]
@andymc12 I saw your PR, which does not use the parent pom. One of the goals is to move to parent pom.xml to control the plugins. Will you be able to adopt the parent pom.xml? I don't think updating checkstyle is show stopper though. We have to do it at one point. Now it's a good time, I think.

**Comment by rdebusscher** [comment]
Correct.  Sorry, I was a bit too fast in replying that Jetty 11 could be used as the restricting factor is WireMock.
